src/patches/widescreen.cpp

```diff
@@ -104,7 +104,7 @@ SMS_WRITE_32(SMS_PORT_REGION(0x802B8B9C, 0x802B0B6C, 0, 0), 0xEC010032);
 static void scaleFOVYPerspectiveMatrix(Mtx mtx, f32 fovY, f32 aspect, f32 nearZ, f32 farZ)
 {
     CPolarSubCamera *cam = gpCamera;
-    reinterpret_cast<f32 *>(cam)[0x48 / 4] = getRecalculatedFovyAngleInc(reinterpret_cast<f32 *>(cam)[0x48 / 4]);
+    reinterpret_cast<f32 *>(cam)[0x48 / 4] = getRecalculatedFovyAngleInc(fovY);
```

Comment:
Change `reinterpret_cast<f32 *>(cam)[0x48 / 4]` to `cam->mProjectionFovy`

Reply:
Fixed.